src/Symfony/Component/Validator/Tests/Fixtures/FakeMetadataFactory.php

```diff
@@ -62,6 +62,8 @@ public function hasMetadataFor($class): bool
     public function addMetadata($metadata)
     {
         $this->metadatas[$metadata->getClassName()] = $metadata;
+
+        return $this;
```

Comment:
Unrelated change? if not, you should add the phpdoc block too: `@return $this`.

Reply:
Used it for a test but not the case anymore, I just revert this change.